Mlem/Window.swift

```diff
@@ -26,7 +27,17 @@ struct Window: View {
     var body: some View {
         content
             .id(appState.currentActiveAccount?.id ?? 0)
-            .onChange(of: flow) { _ in flowDidChange() }
+            .onChange(of: flow) { [flow] _ in
+                switch flow {
+                case let .account(account):
+                    if accountsTracker.savedAccounts.contains(account) {
+                        accountsTracker.update(with: SavedAccount(from: account, lastUsed: .now))
+                    }
+                default:
+                    break
+                }
```

Comment:
Can this just be rolled into `flowDidChange()`? That already has a `switch` on `flow`

Reply:
This code sets the `lastUsed` time for the account that you're *leaving* when you switch accounts. `flowDidChange` is called when the app opens, too (which I suppose is a little misleading), and this code doesn't need to run in that case. Hence why the code is here. 

Putting it in `flowDidChange` would require us to add an `oldValue` parameter to the function, which would have to be `nil` when `flowDidChange` is called from `onAppear`. It could be done, if you'd prefer it that way. An alternative would be to rearrange the logic such that `flowDidChange` is *only* called when the flow changes, and logic that is shared between opening the app and switching accounts can be moved to a `setupFlow` function or something.

Reply:
Ah, gotcha. This seems fine then.